recipe/meta.yaml

```diff
@@ -1,43 +1,55 @@
-{% set version = "3.1.0rc0" %}
+{% set version = "3.2.0rc0" %}
 
 package:
   name: iris
   version: {{ version }}
 
 source:
   url: https://github.com/SciTools/iris/archive/v{{ version }}.tar.gz
-  sha256: d4cf6b827aa15fa6f93078143cd865111e12600b7c68ebfe84e4022c7f3dbfe8
+  sha256: 03589b29d8e20510614ba5d8cf57ce1fca4cf2b128033180beee7e6bf9def202
 
 build:
+  noarch: python
   number: 0
-  skip: true  # [py<37]
 
-requirements:
-  host:
-    - python
-    - pip
-  run:
-    - python
-    - cartopy >=0.18
-    - cf-units
-    - cftime >=1.5.0
-    - dask >=2
-    - matplotlib-base
-    - netcdf4 >=1.4
-    - numpy >=1.14,!=1.15.2
-    - scipy
-    - python-xxhash
+outputs:
+  - name: iris
+    script: build_base.sh
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+      run:
+        - python >=3.8
+        - cartopy >=0.20
+        - cf-units >=3
+        - cftime >=1.5.0 
+        - dask-core >=2
+        - matplotlib-base
+        - netcdf4 >=1.4
+        - numpy >=1.19
+        - scipy
+        - python-xxhash
 
-test:
-  imports:
-    - iris
+    test:
+      imports:
+        - iris
 
+  - name: scitools-iris
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - {{ pin_subpackage('iris', max_pin="x.x.x") }}
+    test:
+      imports:
+        - iris
 about:
```

Comment:
@lbdreyer What's the advantage of doing this?

Reply:
Isn't that what [lines 34-36](https://github.com/conda-forge/iris-feedstock/pull/78/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR34-R36) are doing?

Reply:
My interpretation was that they were separate builds, so they each needed testing (this was originally done in https://github.com/conda-forge/iris-feedstock/pull/74/files).

I will have to do a bit more reading to check this though!

Reply:
Maybe not be helpful, but I found some other examples of where there is a test for each build output:
https://github.com/conda-forge/typing_extensions-feedstock/blob/master/recipe/meta.yaml#L28-L40
and 
https://github.com/conda-forge/gstreamer-feedstock/blob/master/recipe/meta.yaml#L49

Reply:
@ocefpaf Do you have any guidance/opinion on this?

Reply:
No strong opinion but I'd keep the extra test b/c it ensure the multiple output is working.